pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://github.com/HomeXLabs/api_utils
 repository: https://github.com/HomeXLabs/api_utils
```

Comment:
On line 3 of this file, update the version of the library as the null migration document suggests.

Reply:
The changelog will need to be updated as well.